addon/serializers/children.js

```diff
@@ -0,0 +1,12 @@
+import OsfSerializer from './osf-serializer';
+
+export default OsfSerializer.extend({
+    serialize: function(snapshot, options) {
```

Comment:
Perhaps `payloadKeyFromModelName` is what you're looking for to set the type for an individual item?
http://emberjs.com/api/data/classes/DS.RESTSerializer.html#method_payloadKeyFromModelName

Also, try not to duplicate code defined in the parent (eg deleting the relationships section). You can use `this._super(...arguments)` to do things already defined in the parent method.